FAQ.md

```diff
@@ -0,0 +1,26 @@
+# gP2S F.A.Q.s - Frequently Asked Questions
+
+## Usage
+
+#### I created proteins, but no sample is showing up when I try to create a grid
+
+![](FAQ_media/grid_create_no_samples.png)
+
+You have created 2 proteins, but no sample yet.
+
+A sample is a mixture of proteins and ligands (can be 0 proteins or 0 ligands). Once you have created a sample, it will show up in the drop down during grid creation.
```

Comment:
@arohou I belive having at least one component in the sample is mandatory: either protein or ligand.